frontend/components/ResourceCreationPage.tsx

```diff
@@ -109,8 +110,8 @@ const ResourceCreationPage = ({
             <Link href={DIRECTORY_ROUTE} as={DIRECTORY_ROUTE}>
               <a>directory page</a>
             </Link>
-            . If your {OBJECT_NAME_SINGULAR} already exists, please email{' '}
-            <Contact /> to gain access instead of filling out this form.
+            . If your {OBJECT_NAME_SINGULAR} already exists on Hub@Penn, please
```

Comment:
@snie11 Ideally use `SITE_NAME` here in case we ever also use that page for Penn Clubs.